packages/client/docusaurus/docs/client/02-guides/01-authentication.mdx

```diff
@@ -3,8 +3,296 @@ id: 01-authentication
 title: Authenticating Your Users
 ---
 
-Stay tuned for more information on this topic.
+import SDKSpecific from '../SDKSpecific';
 
-[//]: # '- Authenticating Your Users'
-[//]: # '  - video/audio'
-[//]: # '  - with chat'
+## Introduction
+
+Stream uses JWT (JSON Web Tokens) for user authentication in our services.
+These tokens are valid across both Stream Video and Stream Chat services with the assumption
+that your App ID has both Chat and Video enabled.
+
+For security reasons, we recommend that you generate the JWT on your backend server as generating
+them on the client-side can lead to security issues.
```

Comment:
Would make sense to link to more advanced tutorials here. At least to the server-side integration pages (that we'll have in the basics section).

Reply:
Yes, this makes total sense. I guess we can add such links once those server-side pages exist.

Reply:
added.